src/main/java/core/basesyntax/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import java.util.Optional;
+
+public class ReturnOperationHandler implements OperationHandler {
```

Comment:
The class implementation should not begin with an empty line. Please remove the redundant empty line at the beginning of the class definition.